### PR TITLE
fix: make accept header consistent with sync vers.

### PR DESCRIPTION
**Pull request by mh03r932**
Make the accept and content-type header consistent with the sync version

This should fix issue OpenAPITools#6821

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) beforehand.
- [x] Run the shell script `./bin/generate-samples.sh`to update all Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. These must match the expectations made by your contribution. You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/config/java*`. For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`
- [ ] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### CODE REVIEW

**Comment by wing328** [comment]
Please run `./bin/generate-samples.sh ./bin/configs/csharp-netcore-OpenAPIClient` to update the samples (install Git bash if you're using Windows)

cc @mandrean (2017/08) @frankyjuang (2019/09) @shibayan (2020/02)

**Comment by mh03r932** [comment]
Done.
Sorry I could not do this before since I was using githubs online editor.

FYI I forced pushed that because I made a C&P error and wanted to keep this PR to a single commit.

File: modules/openapi-generator/src/main/resources/csharp-netcore/api.mustache [hunk 1]
```diff
@@ -418,12 +418,13 @@ namespace {{packageName}}.{{apiPackage}}
...
-            
-            foreach (var _accept in _accepts)
-                localVarRequestOptions.HeaderParameters.Add("Accept", _accept);
+
+
+			var localVarContentType = {{packageName}}.Client.ClientUtils.SelectHeaderContentType(_contentTypes);
+			if (localVarContentType != null) localVarRequestOptions.HeaderParameters.Add("Content-Type", localVarContentType);
+
+			var localVarAccept = {{packageName}}.Client.ClientUtils.SelectHeaderAccept(_accepts);
+			if (localVarAccept != null) localVarRequestOptions.HeaderParameters.Add("Accept", localVarAccept);
```
Comment (shibayan) [inline modules/openapi-generator/src/main/resources/csharp-netcore/api.mustache:427]: Modify from tabs to spaces to match the indentation of the existing code.

**Comment by wing328** [comment]
CircleCI failure already fixed in the master.

**Pull request merged**